include/wx/event.h

```diff
@@ -2725,6 +2727,30 @@ class WXDLLIMPEXP_CORE wxMaximizeEvent : public wxEvent
     wxDECLARE_DYNAMIC_CLASS_NO_ASSIGN(wxMaximizeEvent);
 };
 
+/*
+ wxEVT_FULLSCREEN
+ */
+class WXDLLIMPEXP_CORE wxFullScreenEvent : public wxEvent
+{
+public:
+    wxFullScreenEvent(int winid = 0, bool fullscreen = true)
+        : wxEvent(winid, wxEVT_FULLSCREEN)
+        { m_fullscreen = fullscreen; }
+    wxFullScreenEvent(const wxFullScreenEvent& event)
+        : wxEvent(event)
+    { m_fullscreen = event.m_fullscreen; }
```

Comment:
```suggestion
        { m_fullscreen = event.m_fullscreen; }
```
(just for consistency with the line above; alternatively that one could be unindented too, of course)